src/discussions/discussions-home/FeedbackWrapper.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect } from 'react';
+
+import { useSelector } from 'react-redux';
+
+import { logError } from '@edx/frontend-platform/logging';
+
+import { RequestStatus } from '../../data/constants';
+import {
+  selectconfigLoadingStatus,
+  selectIsCourseAdmin,
+  selectIsCourseStaff,
+  selectUserIsGroupTa,
+  selectUserIsStaff,
+} from '../data/selectors';
+
+export default function useFeedbackWrapper() {
+  const isStaff = useSelector(selectUserIsStaff);
+  const isUserGroupTA = useSelector(selectUserIsGroupTa);
+  const isCourseAdmin = useSelector(selectIsCourseAdmin);
+  const isCourseStaff = useSelector(selectIsCourseStaff);
```

Comment:
util function can be made for all these roles and then used below in if condition

Reply:
I can create a new selector like this but it would be overkill and will not be used anywhere else.
So I guess we should go ahead without adding a new selector.

```js
const selectUserPermissions = createSelector(
  [selectUserHasModerationPrivileges, selectUserIsStaff, selectUserIsGroupTa],
  (hasModerationPrivileges, isUserAdmin, isGroupTa) => {
    return hasModerationPrivileges || isUserAdmin || isGroupTa;
  }
);
```